templates/_composition/MVVMLight/Project/Services/ActivationService_postaction.cs

```diff
@@ -5,13 +5,8 @@ internal class ActivationService
         private readonly Type _defaultNavItem;
 //{[{
 
-        private NavigationServiceEx NavigationService
-        {
-            get
-            {
-                return Microsoft.Practices.ServiceLocation.ServiceLocator.Current.GetInstance<NavigationServiceEx>();
-            }
-        }
+        private ViewModels.ViewModelLocator Locator => Application.Current.Resources["Locator"] as ViewModels.ViewModelLocator;
+        private NavigationServiceEx NavigationService => Locator.NavigationService;
```

Comment:
NavigationService must be named NavigationService because in this case, this postaction add a local variable thath in other frameworks will be a static class. I've added Locator instead of locator to be more consistent with NavigationService.

Reply:
so then adjust the variables to public and match the other framework.  as is, privates should be _foo, not Foo.  this is about good generated code.  We should be consistent through out our generated code.